### DISCUSSION TITLE
Can chaste be compiled on clang for llvm

### DISCUSSION THREAD

**Discussion by Elphikaus** [Chaste/Chaste#165] [Q&A] [answered]
As far as I know, there is still an issue with chaste on docker on ARM M series processors. Essential an ARM is emulated, which is really slow.

Is it possible or has it been done to start compiling chaste with clang and take advantage of the LLVM then the machine architecture is sorted.

Look at Julia and Rust as LLVM examples, there is minimal issue with machine output.

**Comment by Elphikaus** [comment]
I will give it a try when I get a chance, but thought I would throw it out there.

**Comment by bdevans** [comment, marked as answer]
Hi, have you tried the latest `chaste/develop` docker image? These are built for both `amd` and `arm` processors. Is it still slow when you try it?

**Reply by Elphikaus** [reply]
I have, but I get an error when

Once I open the docker container 

cd lib
cmake ~/src

I get the error in the issue below. 

See

https://github.com/Chaste/chaste-docker/issues/15
